week-2/G-conditionals-2/exercise-1.js

```diff
@@ -7,9 +7,16 @@
 */
 
 function negativeOrPositive(number) {
-
+  if (number>=5){
```

Comment:
Do you need to check the value `5` here? Remember that the definition of a function should work for all its inputs, rather than just the calls we are using below to check it. We are calling `negativeOrPositive(5)` but it could easily be any other number (e.g `negativeOrPositive(4)`).